compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1354,6 +1420,18 @@ getMatmulVectorSizes(mlir::FunctionOpInterface entryPointFn,
     }
   }
 
+  if (isRISCV(targetAttr) && hasAnyVFeature(targetAttr)) {
+    bool isMTB = isa<linalg::MatmulTransposeBOp>(op);
+    bool isBMTB = isa<linalg::BatchMatmulTransposeBOp>(op);
+    // Use default tile size for matmul_transpose_b &
+    // batch_matmul_transpose_b to avoid performance drop.
+    if (!isMTB && !isBMTB) {
```

Comment:
nit: you can collapse it to one `isa` in the if condition.

```suggestion
    // Use default tile size for matmul_transpose_b &
    // batch_matmul_transpose_b to avoid performance drop.
    if (!isa<linalg::MatmulTransposeBOp, linalg::BatchMatmulTransposeBOp>(op)) {
```

Reply:
Applied suggestion.